src/actions/breakpoints/breakpointPositions.js

```diff
@@ -44,10 +47,18 @@ function convertToList(results, sourceId) {
 export function setBreakpointPositions(location: SourceLocation) {
   return async ({ getState, dispatch, client, sourceMaps }: ThunkArgs) => {
     let sourceId = location.sourceId;
+
     if (getBreakpointPositionsForSource(getState(), sourceId)) {
-      return;
+      return getBreakpointPositionsForSource(getState(), sourceId);
```

Comment:
Calling this twice seems unnecessary, can we save the result the first time instead?